packages/sdk/src/types/common.ts

```diff
@@ -73,11 +73,6 @@ export interface SignTransction{
   statusCallback?: (result: ISubmittableResult) => void;
 }
 
-export interface FetchRoles{
```

Comment:
Already have an interface called ResponseFetchGroup that has these properties

Reply:
Keep FetchRoles and remove ResponseFetchGroup if that's the case

Reply:
Thanks for clarification, I missed this. I will create a new object called ResponseFetchRole to represent it better.